bitnami/grafana/templates/deployment.yaml

```
@@ -316,6 +323,11 @@ spec:
             secretName: {{ .Values.config.grafanaIniSecret }}
           {{- end }}
         {{- end }}
+        {{- if .Values.ldap.tls.certificatesSecret }}
+        - name: ldap-tls
+          secret:
+            secretName: {{ .Values.ldap.tls.certificatesSecret }}
+        {{- end }}
```

The condition for the volumeMount is `.Values.ldap.tls.enabled`, so I think we could use the same condition here.

What do you think about this:

```suggestion
        {{- if .Values.ldap.tls.enabled }}
        - name: ldap-tls
          secret:
            secretName: {{ required "Please provide a secret containing TLS certificates to enable 'ldap.tls.certificatesSecret'" .Values.ldap.tls.certificatesSecret }}
        {{- end }}
```

---

Thanks for pointing out, there was a failure there. 

About your suggestion, the certificates are not required when `ldap.tls` is enabled. If the LDAP certificate is signed by a wide trusted authority or the user disabled any validation with `ldap.extraConfiguration` , `tls.certificatesSecret` won't be required